ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3175,10 +3175,18 @@ literals:
   - ACTIVE
   - INACTIVE
 
+- up_shade_command:
+  - ON
+  - OFF
+- down_shade_command:
```

Comment:
Re-name to `shade_down_command` after moving the subfields to the `measurement_descriptor` category

Reply:
done